src/core/actions/assert.js

```diff
@@ -213,7 +213,7 @@ const checkElementAvailability = async (state, { isAvailable }, { args: { select
           result = await browser.execute(
             findElements,
             selectorText,
-            undefined,
+            marker,
```

Comment:
`marker` text might contain regex metacharacters. Please use `buildRegexEscapedString` to regex-escape `marker` string. Please check `checkElementStatus` for an example usage of `buildRegexEscapedString`.

```suggestion
            buildRegexEscapedString(marker),
```

Reply:
Addressed. Could you check now ?